raiden-ts/package.json

```diff
@@ -128,75 +120,13 @@
     "/dist",
     "/dist:cjs"
   ],
-  "eslintConfig": {
-    "parser": "@typescript-eslint/parser",
-    "plugins": [
-      "@typescript-eslint",
-      "jsdoc"
-    ],
-    "extends": [
-      "plugin:@typescript-eslint/recommended",
-      "prettier/@typescript-eslint",
-      "plugin:prettier/recommended",
-      "plugin:jsdoc/recommended",
-      "plugin:import/errors",
-      "plugin:import/warnings",
-      "plugin:import/typescript"
-    ],
-    "env": {
-      "browser": true,
-      "es6": true,
-      "node": true
-    },
-    "globals": {
-      "Atomics": "readonly",
-      "SharedArrayBuffer": "readonly"
-    },
-    "parserOptions": {
-      "ecmaVersion": 2018,
-      "sourceType": "module"
-    },
-    "rules": {
-      "import/order": [
-        "error"
-      ],
-      "import/export": "off",
-      "@typescript-eslint/explicit-module-boundary-types": "off",
-      "@typescript-eslint/explicit-function-return-type": "off",
-      "@typescript-eslint/no-non-null-assertion": "off",
-      "@typescript-eslint/no-empty-interface": "off",
-      "@typescript-eslint/no-unused-vars": [
-        "warn",
-        {
-          "varsIgnorePattern": "^_"
-        }
-      ],
-      "jsdoc/no-types": "warn",
-      "jsdoc/require-param-type": "off",
-      "jsdoc/require-returns-type": "off",
-      "jsdoc/require-hyphen-before-param-description": "warn",
-      "jsdoc/require-jsdoc": [
-        1,
-        {
-          "publicOnly": true
-        }
-      ]
-    }
-  },
   "eslintIgnore": [
```

Comment:
As we have discussed already, I think it is cleaner to put this list into a ignore file.

Reply:
Sure. May you do it, please, so we don't conflict? Thanks

Reply:
:heavy_check_mark: